features-json/fullscreen.json

```diff
@@ -413,7 +413,7 @@
       "14.5-14.8":"a #5",
       "15.0-15.1":"n",
```

Comment:
Not directly related: I noticed Full Screen API was `n` for Safari on iOS 15.x before ([initial change](https://github.com/Fyrd/caniuse/commit/725d2b36644ac1dc593021f9b00b72ecc5328c06#diff-15d3af05bdeb0fe2ab82eac9b0103fa19eeb5eff2cf49ba2b5b9ba7c0b07f001) by @Fyrd) - I compared 14 and 15 on iPad using browserstack, seemed the same for me. I can send another PR for that after this one :)

Reply:
I'm conferring with engineers to verify the correct data here.

Reply:
I verified that 15.0-15.4 should retain the same partial support as the 14.5-14.8 entry.